rna_design/constraint_extractor.py

```diff
@@ -5,7 +5,27 @@
 
 
 class ConstraintExtractor():
+    """
```

Comment:
_extract_constraint_string should be renamed to
_extract_sequence_constraints
and _extract_dot_struct should be renamed to
_extract_structure_constraints